voskcli/transcribe.py

```diff
@@ -79,6 +102,19 @@ def write_captions_paragraph(vtt, paragraph):
 
 
 def write_webvtt_captions(rec_results):
+    '''
+    Process transcription data.
+    
+    Split transcription data into lines of up to 35 chars and paragraphs
+    of up to 2 lines (based on MAX_CHARS_PER_LINE and MAX_LINES_IN_PARAGRAPH),
+    and call `write_captions_paragraph` function to append paragraphs to vtt file.
```

Comment:
mentioning of function call maybe not needed. Could be rephrased as simply `and append paragraphs to vtt file`. Same goes for `transcription` and `main` functions.

Reply:
I usually wouldn't document inner function calls, but I also don't mind if you think these are important/relevant to someone using this.